site/docs/release-notes/3-11.md

```diff
@@ -50,11 +50,15 @@
 
 ??? functional "Additional HTTP headers can now be passed to security plugins"
 
-    Additional HTTP headers can now be passed to security plugins, and will also be propogated on federated queries
+    Additional HTTP headers can now be passed to security plugins, and will also be propogated on federated queries.
 
-    See [odpi/egeria#5700](https://github.com/odpi/egeria/issues/5700){ target=gh } for the implementation.
+    See [odpi/egeria#5700](https://github.com/odpi/egeria/issues/5700){ target=gh } for the implementation discussion.
+
+    How it works:
+
+    * When header.name.list is set, the HttpRequestHeadersFilter puts all specified headers in a ThreadLocal variable.
+    * ThreadLocal ensures headers consistency between different requests (threads). The HttpHeadersThreadLocal class which holds the ThreadLocal is in the http-helper module; therefore this functionality can be used in any module that includes http-helper.
```

Comment:
As this is implementation detail, I wonder if we need it in the release notes? It's useful info for sure, really belongs with detail on writing a security plugin? 

I don't think it's a problem having it, just a thought

Additionally this content needs to form part of the main docs really, not just as a release note. I think we need a section related to writing a security connector

That probably includes
 - https://github.com/odpi/egeria-docs/blob/267e85a35adc2b133e4b8b97895c199e152f7272/site/docs/guides/admin/servers/configuring-the-server-security-connector.md
 - https://github.com/odpi/egeria-docs/blob/267e85a35adc2b133e4b8b97895c199e152f7272/site/docs/guides/admin/servers/configuring-the-server-security-connector.md
 
 I think it's ok to just add the release notes in a seperate PR if that's easier - getting out something quickly since it's in the release, but I do think it's important we have this update too

Reply:
@planetf1 I modified the release notes text and added the technical details on the configuration page. Please have a look and tell me if you have any suggestions

Reply:
I wonder if this should be added to the current release notes - 3.14 or 3.15. Though the change was made much earlier, readers may only view release notes for new releases?